### DISCUSSION TITLE
改善连接速度的想法

### DISCUSSION THREAD

**Discussion by xfgryujk** [xfgryujk/blivechat#235] [想法]
* 前端放在Vercel，应该比CF速度快点，没这么容易被墙
* 后端支持多个endpoint，某个不可用时可以做故障转移
* 启动时做服务发现，方便改endpoint
* 一个后端裸连，一个放在CF后面，万一被D到黑洞了至少保证CF的能用

**Comment by xfgryujk** [comment]
这么快就被打脸了，Vercel我也裸连不上了

**Comment by SSSSSSSan** [comment]
我用api网关高枕无忧
